steward/src/cellars.rs

```diff
@@ -204,6 +219,15 @@ pub fn validate_oracle(
     Ok(())
 }
 
+pub fn validate_cache_price_router(cellar_id: &str) -> Result<(), Error> {
```

Comment:
Sun mentioned we were expecting `true` and `500` for the args. Should we validate against that?

Reply:
I can, do we usually validate non-identity arguments?